pkg/execution/batch/redis.go

```diff
@@ -33,6 +37,43 @@ type redisBatchManager struct {
 	q redis_state.QueueManager
 }
 
+func (b redisBatchManager) batchKey(ctx context.Context, evt event.Event, fn inngest.Function) (string, error) {
+	if fn.EventBatch.Key == nil {
+		return fn.ID.String(), nil
```

Comment:
sorry, I just realized that `BatchPointer` is already namespace with the functionID.
https://github.com/inngest/inngest/blob/16f8e28ddcb6eca3b28df3319b45f36a7ba363c6/pkg/execution/state/redis_state/key_generator.go#L324-L326

a fnID string could be redundant, so a `default` string might be enough in this case. not a big deal as long as the fallback string is consistent.

Reply:
Good point, changed to `default`